crates/tap-agent/src/agent/sender_allocation.rs

```diff
@@ -934,6 +935,7 @@ pub mod tests {
         pgpool: PgPool,
         sender_aggregator_endpoint: String,
         escrow_subgraph_endpoint: &str,
+        rav_request_receipt_limit: u64,
         sender_account: Option<ActorRef<SenderAccountMessage>>,
     ) -> SenderAllocationArgs {
```

Comment:
This may be a little out of the scope, but one way to make this better is to add `bon` and annotate this function with `#[bon::builder]`. You are able to set default and only provide member that you actually need to change.

https://bon-rs.com/

Reply:
Feel free to use it or not, it's your choice.

Reply:
Oh yes, I did liked this bon thing

Reply:
Hey, you didn't use bon here. Add it and provide a default value.
```suggestion
        pgpool: PgPool,
        sender_aggregator_endpoint: String,
        escrow_subgraph_endpoint: &str,
         #[builder(default = 1000)] rav_request_receipt_limit: u64,
        sender_account: Option<ActorRef<SenderAccountMessage>>,
    ) -> SenderAllocationArgs {
```

Reply:
Hey, no updates here?